conf/nginx.conf

```diff
@@ -1,37 +1,32 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 # https://github.com/WeblateOrg/weblate/blob/master/weblate/examples/weblate.nginx.conf
-location = __PATH__/favicon.ico {
-  # DATA_DIR/static/favicon.ico
-  alias __FINALPATH__/data/static/favicon.ico;
-  expires 30d;
-}
+    location ~ ^__PATH__/favicon.ico$ {
 
-location = __PATH__/robots.txt {
-  # DATA_DIR/static/robots.txt
-  alias __FINALPATH__/data/static/robots.txt;
-  expires 30d;
-}
+        alias __INSTALL_DIR__/data/cache/static/favicon.ico;
+        expires 30d;
+    }
 
```

Comment:
Did you remove robots.txt ?

Reply:
Yes because I applied the default weblate.conf and there is no robot.txt in the directory

Reply:
@kay0u any opinion?